index.bs

```diff
@@ -2537,13 +2537,15 @@ Short Circuit</h5>
 	and potential members of a Council who are not renouncing their seat
 	confirm it by a vote which results in both of the following:
 	* at least 80% of them vote affirmatively to adopt this resolution
+      (in addition, Tim Berners-Lee <em class=rfc2119>may</em> also abstain)
```

Comment:
"In addition ... also" is odd phrasing. This might work --
```suggestion
	* Tim Berners-Lee either abstains or votes affirmatively to adopt this resolution
```

Reply:
Or this, with a bit of future proofing, even if it's an unlikely future, and making it a little less Timbl-specific (even if that is the underlying intent) —
```suggestion
	* Directors Emeriti either abstain or vote affirmatively to adopt this resolution
```

Reply:
and if he votes no? I am unclear as to what is trying to be achieved here. If it's that it's 80% of the body without Tim, then say that Tim does not count towards quorum, but may voyte, abstain, or fail to vote.

Reply:
I presume, if he votes "no", then the last bullet is triggered (i.e., one of "the potential members of the Council who did not renounce their seat" voted against adopting the resolution), and the resolution fails.

(As I understand this, there's an intent that only Directors Emeriti can submit an abstention; the other potential Council members can only abstain by not submitting a ballot.)

```suggestion
	* Directors Emeriti may record a vote of abstention, unlike other potential members of the Council, who can only abstain by not voting
```

Clearer formulation may require a more complete rephrasing.

Reply:
> As I understand this, there's an intent that only Directors Emeriti can submit an abstention; the other potential Council members can only abstain by not submitting a ballot.

No. At least that's not my intent. The original intent was that only Timbl can abstain (by not voting), and everybody else has to vote affirmatively, or the short circuit fails. We have since (in #885) relaxed the rule to say that anybody can abstain (by not voting) as long as it's not too many people doing so. I believe this makes this special rule about Timbl redundant and unnecessary, and that we should close this PR.

@tantek has said that he thinks that even with the adoption of #885, he still thinks this PR is useful, which is why I have rebased it on top of the text that allows for abstention (as long as we stay above 80%). The awkwardness of adding in the Timbl clause comforts me in the idea that this is not helpful.